### DISCUSSION TITLE
Can't access osu! website on Firefox?

### DISCUSSION THREAD

**Discussion by amchoudhry** [ppy/osu#18112] [Q&A]
So this has been happening for a week and it's giving me this text. it says that "osu.ppy.sh has a security policy called HTTP Strict Transport Security (HSTS), which means that Firefox can only connect to it securely. You can’t add an exception to visit this site."

```
https://osu.ppy.sh/community/forums/topics/776267

The certificate is not trusted because it is self-signed.

HTTP Strict Transport Security: true
HTTP Public Key Pinning: false

Certificate chain:

-----BEGIN CERTIFICATE-----
MIIDYzCCAkugAwIBAgIJAMwT2udpny5NMA0GCSqGSIb3DQEBCwUAMFIxCzAJBgNV
BAYTAlVTMQ0wCwYDVQQIDARDYXJ5MQ0wCwYDVQQHDARDYXJ5MRIwEAYDVQQKDAlv
c3VnYXRhcmkxETAPBgNVBAMMCCoucHB5LnNoMB4XDTE4MTAxNjAwNTcwMVoXDTI4
MTAxMzAwNTcwMVowUjELMAkGA1UEBhMCVVMxDTALBgNVBAgMBENhcnkxDTALBgNV
BAcMBENhcnkxEjAQBgNVBAoMCW9zdWdhdGFyaTERMA8GA1UEAwwIKi5wcHkuc2gw
ggEiMA0GCSqGSIb3DQEBAQUAA4IBDwAwggEKAoIBAQDIf7UgcX69O3Ku4+tRGP9P
ntXZNgg0UzEX6P5VwOtGQcwQj5ba25lbBVXTYanMiEwtRTYzoFd5V6Y/cq9ySiOF
MhToKtQcSNJJWyU81VcUWiQRY3cJvebqaWdZFVt1KiT77hj/5vOzv79+yj8lbkf1
zGFExA4Y3fXPoP5k/3IlYOHz0kMLj/QyC/hHxp3LtRjUJBhqi/KBod+ue7SCOMvo
DvZe5SwRjBC89sr5wfuzvbNC6E+ZLn3LS84TFY9m9zqBK/xDygKMfMrNmrESkWZq
jIbbCXxh0pZ8Nq3sZ9mEE3i04lkk69s8ZFe0/t2p/ocdgb/4vH6GgZzL7LPMrG1L
AgMBAAGjPDA6MA4GA1UdDwEB/wQEAwIDiDATBgNVHSUEDDAKBggrBgEFBQcDATAT
BgNVHREEDDAKgggqLnBweS5zaDANBgkqhkiG9w0BAQsFAAOCAQEAYSmTCdJNy2Zc
+BXaViDvPO+yFQ7/uKQ5ThMqQNbZoYnsPBuupMUPVYzTN2xrtFXMXQAt1UJUK0sP
LKkYEpOFFXPsiwPKYTQTJlgs7okflesfQy0/Di0Ex2Ez5KyN5l3LUZ14OFwJn8A7
K7EctyrBkmIUWkK7VJSqWzy61iuLPMesW1CccEHjn+XLLCSAI/3Q8LThmB9IHJM3
OeJfsZXSaJVZen5qin7euTd58Sw/dtONootwKiCxRe+uo7tVWb3+CI6YN1w4wwmv
YoUFi6k6jsfQj9QE3Wp4Gl41owDNe+b74YIjxqWsyJ0VQ9zLTrLmk0SRs1oXNHIP
n+4ZE3fmrQ==
-----END CERTIFICATE-----
```

Any help on how I can access osu! website

**Comment by bdach** [comment]
This is not the right place to report this, for one, but more importantly, a self-signed certificate should not be anywhere near getting used when connecting to the website. On my end (also use firefox), I see that the certificate used is verified by two CAs:

![image](https://user-images.githubusercontent.com/20418176/167035239-c1bb7238-6958-4d1e-bb68-40af019912df.png)

Are you trying to connect from a work or school computer that could be managed by the work/school's IT department?

**Reply by smoogipoo** [reply]
Could also be some antivirus MITMing... Have seen this before with their "web protection" services.
